jina/clients/__init__.py

```diff
@@ -169,13 +172,46 @@ def delete(
             self._get_results, inputs, on_done, on_error, on_always, **kwargs
         )
 
+    def dump(
+        self,
+        path: str,
+        on_done: CallbackFnType = None,
+        on_error: CallbackFnType = None,
+        on_always: CallbackFnType = None,
+        shards: int = None,
+        formats: List[DumpTypes] = None,
```

Comment:
I would find the interface cleaner, if that would be just `format` and if you want to export to different formats, you need to call multiple times. I see the point, that if you export to multiple formats in one go, it may be easier to optimize. But I don't see that use-case as predominant and thus, would prefer a cleaner interface.

Reply:
The format code now went elsewhere. Question remains the same.

Reply:
I thought we agreed to remove formats from the MVP, and just dump to default. We would then add this at another point, if we go that way.

Reply:
:+1: